app/views/schools/add_participants/confirm.html.erb

```diff
@@ -55,9 +57,10 @@
             </div>
           </dd>
           <dd class="govuk-summary-list__actions">
-            <% unless add_participant_form.type == :self %>
-              <%= govuk_link_to({ action: :show, step: "details" }) do %>
-                Change <span class="govuk-visually-hidden">personal details</span>
+            <% unless add_participant_form.type == :self || add_participant_form.mentor_options.blank? %>
```

Comment:
Could we change it to `if` - negated negation are a bit tricky to read. Also, could we replace `blank?` with `any?` or `none?` if negated (this will trigger `EXISTS` instead of `SELECT` and is slightly better semantically)